src/run.js

```diff
@@ -0,0 +1,116 @@
+const cozyClient = require('cozy-konnector-libs/dist/libs/cozyclient')
+const errors = require('cozy-konnector-libs/dist/helpers/errors')
+const { log } = require('cozy-konnector-libs')
+
+const {
+  fetchAndSaveTrips,
+  fetchAndSaveManualEntries,
+  fetchTripsMetadata
+} = require('./lib')
+const { getFirstAndLastTripTimestamp } = require('./trace-requests')
+const {
+  canSaveNextTripsChunk,
+  restartKonnector,
+  createChunks
+} = require('./utils')
+const { saveAccountData } = require('./save')
+const { TRIPS_CHUNK_SIZE } = require('./const')
+const client = cozyClient.new
+
+const getTimeout = () => {
+  const maxExecutionTimeSeconds = parseInt(process.env.COZY_TIME_LIMIT, 10)
+  return maxExecutionTimeSeconds - 100
```

Comment:
100 is arbitrary? Since you already created a lot of constant having this one would be great 👍